src/SFA.DAS.EmployerAccounts.Web.UnitTests/Controllers/EmployerTeamControllerTests/WhenNoPayeScheme.cs

```diff
@@ -49,11 +49,7 @@ public void ThenTheAddPayeViewIsReturnedAtRow1Panel1()
         {
             // Arrange
             var model = new AccountDashboardViewModel();
-            model.PayeSchemeCount = 0;
-            model.CallToActionViewModel = new CallToActionViewModel
-            {
```

Comment:
Why is CallToActionViewModel removed?

Reply:
ApprenticeshipAdded  removed from the ViewModel, hence removed